pages/auth/register.tsx

```diff
@@ -1,54 +1,128 @@
+import axios from "axios";
+import * as yup from "yup";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+import { useRouter } from "next/router";
 import { Button } from "flowbite-react";
 import { UserIcon } from "../../components/icons";
 import { AuthLayout } from "../../components/layout";
 import { RadioButton, TextInput } from "../../components/ui";
 
 const RegisterPage = () => {
+  let schema = yup.object().shape({
```

Comment:
we should probably have an eslint rule that complains about not using `let` when the variable is never reassigned